docs/source/common_api_flows.rst

```diff
@@ -2,66 +2,66 @@ Common API workflows
 --------------------
 
 There are many genomics-related APIs documented at
-`cloud.google.com/genomics/v1beta2/reference <https://cloud.google.com/genomics/v1beta2/reference>`_.
+`cloud.google.com/genomics/reference/rest/ <https://cloud.google.com/genomics/reference/rest/>`_.
 
-Of the available calls, there are some very common patterns that can 
-be useful when developing your own code. 
+Of the available calls, there are some very common patterns that can
+be useful when developing your own code.
 
-The following sections describe these workflows using plain URLs and 
+The following sections describe these workflows using plain URLs and
 simplified request bodies. Each step should map 1-1 with all of the auto-generated client libraries.
 
 
 Browsing read data
 ~~~~~~~~~~~~~~~~~~
 
-* ``GET /datasets``                                    
-  
+* ``GET /datasets``
+
   List all available datasets that a current user has access to. (Or all public datasets when not using OAuth)
   Choose one datasetId from the result.
-  
+
   Note: Currently, this call only returns public datasets! It is not able to return any private datasets.
   For now, you may need to ask a user for a datasetId or readGroupSetId directly
 
 * ``POST /readgroupsets/search {datasetIds: [<datasetId>]}``
 
   Search for read group sets in a particular dataset. Choose one readGroupSetId from the result.
-  
-  Note: This is a good place to use a `partial request <https://cloud.google.com/genomics/performance#partial>`_ 
-  to only ask for the id and name fields on a read group set. Then you can follow up with a 
+
+  Note: This is a good place to use a `partial request <https://cloud.google.com/genomics/performance#partial>`_
+  to only ask for the id and name fields on a read group set. Then you can follow up with a
   ``GET /readgroupsets/<readGroupSetId>`` call to get the complete read group set data.
-  
+
 * ``GET /readgroupsets/<readGroupSetId>/coveragebuckets``
-  
-  Get coverage information for a particular readset. This will tell you where the read data is located,
+
+  Get coverage information for a particular read group set. This will tell you where the read data is located,
   and which referenceNames should be used in the next step.
-  
-* ``POST /reads/search {readGroupSetIds: [<readGroupSetId>]}``   
 
-  Get reads for a particular read group set. 
-  
-  Note: The call also requires ``referenceName``, ``start`` and ``end``. 
+* ``POST /reads/search {readGroupSetIds: [<readGroupSetId>]}``
+
+  Get reads for a particular read group set.
+
+  Note: The call also requires ``referenceName``, ``start`` and ``end``.
```

Comment:
Actually it does not seem to require the `referenceName`, `start` and `end` and seems to return data without them.